lib/racecar/rails_config_file_loader.rb

```diff
@@ -5,27 +5,27 @@ def self.load!
 
       begin
         require "rails"
-      rescue LoadError
-        # Not a Rails application.
-      end
 
-      if defined?(Rails)
-        $stderr.puts "=> Detected Rails, booting application..."
+        if defined?(Rails)
+          $stderr.puts "=> Detected Rails, booting application..."
 
-        require "./config/environment"
+          require "./config/environment"
 
-        if (Rails.root + config_file).readable?
-          Racecar.config.load_file(config_file, Rails.env)
-        end
+          if (Rails.root + config_file).readable?
+            Racecar.config.load_file(config_file, Rails.env)
+          end
 
-        # In development, write Rails logs to STDOUT. This mirrors what e.g.
-        # Unicorn does.
-        if Rails.env.development? && defined?(ActiveSupport::Logger)
-          console = ActiveSupport::Logger.new($stdout)
-          console.formatter = Rails.logger.formatter
-          console.level = Rails.logger.level
-          Rails.logger.extend(ActiveSupport::Logger.broadcast(console))
+          # In development, write Rails logs to STDOUT. This mirrors what e.g.
+          # Unicorn does.
+          if Rails.env.development? && defined?(ActiveSupport::Logger)
+            console = ActiveSupport::Logger.new($stdout)
+            console.formatter = Rails.logger.formatter
+            console.level = Rails.logger.level
+            Rails.logger.extend(ActiveSupport::Logger.broadcast(console))
+          end
         end
+      rescue LoadError
+        # Not a Rails application.
```

Comment:
If it's not a rails application then perhaps we should output something like this:
```
 $stderr.puts "=> Detected Ruby, booting application..."
```
Any thoughts regarding that?